sklearn/tests/test_discriminant_analysis.py

```diff
@@ -257,6 +260,33 @@ def test_lda_store_covariance():
     )
 
 
+@pytest.mark.parametrize('n_features', [3, 5])
+@pytest.mark.parametrize('n_classes', [5, 3])
+def test_lda_dimension_warning(n_classes, n_features):
+    RNG = check_random_state(0)
```

Comment:
lowercase please. upper case is reserved for module level constants, right? (yes, X violates that, I know).

Reply:
that's right, will do